webcompat/static/js/lib/models/comment.js

```diff
@@ -25,5 +25,45 @@ issues.Comment = Backbone.Model.extend({
       createdAt: moment(response.created_at).fromNow(),
       rawBody: response.body
     });
+    var linkHeaderRegex = /(?=.*?(link|last|next|comments|page)).*/ 
+    if (linkHeaderRegex.test(jqXHR.xhr.getAllResponseHeaders())) { 
+      response.lastPageNumber = this.parseHeader(jqXHR.xhr.getResponseHeader('Link')).last.split('\?page\=')[1]; 
+    }
+    else if(!jqXHR.xhr.getAllResponseHeaders().includes('link:')){ 
```

Comment:
`String.prototype.includes` is not well enough supported. I would not recommend using ES6 in webcompat right now (not without transpiling back to ES5, which we're not set up to do).